modules/transport-netty4/build.gradle

```diff
@@ -55,15 +59,15 @@ tasks.named("dependencyLicenses").configure {
   mapping from: /netty-.*/, to: 'netty'
 }
 
-test {
+internalClusterTest {
```

Comment:
Dont' we still need this system property for `test`?

Reply:
indeed. https://github.com/elastic/elasticsearch/pull/59939/commits/d5e45728ec18fa707685582cc0db065d45591f59 thanks!